scripts/build/macos/build-client.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd ../../..
+rm -f bin/clientCM.jar
+rm -f bin/coreCM.jar
+./scripts/maven/mvnw -am --projects climate-monitoring-server clean install
```

Comment:
You area building for the server, but in this file you should build for the client